python/pip_install/pip_repository.bzl

```diff
@@ -513,6 +528,22 @@ python_interpreter. An example value: "@python3_x86_64-unknown-linux-gnu//:pytho
     "repo_prefix": attr.string(
         doc = """
 Prefix for the generated packages will be of the form `@<prefix><sanitized-package-name>//...`
+""",
+    ),
+    "target_platforms": attr.string_list(
```

Comment:
Long term, should we just plan on making target_platforms=all the default behavior?

Reply:
Correct.